files/templates/settings2.html

```diff
@@ -85,11 +85,13 @@
 											<i class="fas fa-user-slash pr-2"></i>Permabanned Users
 										</a>
 									</li>
+									{% if v and v.admin_level >= 2 %}
```

Comment:
There's another link to /blocks in the markup for the mobile tab-bar on L127-129, which similarly needs to be wrapped in an admin_level conditional.

Reply:
Thanks, missed that. Done!